phoenix-core/src/main/java/org/apache/phoenix/hbase/index/IndexRegionObserver.java

```diff
@@ -565,12 +829,6 @@ private void prepareIndexMutations(
                   // add the VERIFIED cell, which is the empty cell
                   Mutation m = update.getFirst();
                   if (context.rebuild) {
-                      if (m instanceof Put) {
-                          long ts = getMaxTimestamp(m);
-                          // Remove the empty column prepared by Index codec as we need to change its value
-                          removeEmptyColumn(m, emptyCF, emptyCQ);
-                          ((Put) m).addColumn(emptyCF, emptyCQ, ts, VERIFIED_BYTES);
```

Comment:
Now that this snippet is added in the next commit, the above comment makes sense too. Thanks.